pkg/controller/workload/workload_processor.go

```diff
@@ -494,17 +496,15 @@ func (p *Processor) storeServiceFrontendData(serviceId uint32, service *workload
 	return nil
 }
 
-// toLLb indicates whether we are performing a locality load balance update.
-// If toLLb is true, it means we need to calculate priority; otherwise,
-// it represents a random strategy, in which case we just set the priority to 0.
-func (p *Processor) updateEndpoint(serviceId uint32, toLLb bool) error {
-	var (
-		err  error
-		prio uint32
-	)
-	eksDelete := []bpf.EndpointKey{}
-	backendUids := []uint32{}
-	for _, ep := range p.EndpointCache.List(serviceId) {
+func (p *Processor) updateEndpointOneByOne(serviceId uint32, epsDelete []cache.Endpoint, toLLb bool) error {
```

Comment:
`epsDelete` may look confusing, indeed we are trying to re orgnize them

Reply:
using `epsUpdate` now